### ISSUE TITLE
Show addon icon in address bar

### ISSUE THREAD

**Issue by sheikh-azharuddin** [mozilla-mobile/fenix#17894]
[comment]: # (Please do your best to search for duplicate issues before filing a new issue so we can keep our issue board clean)
[comment]: # (Every issue should have exactly one feature request described in it. Please do not file feedback list tickets as it is difficult to parse them and address their individual points)
[comment]: # (Feature Requests are better when they’re open-ended instead of demanding a specific solution e.g: “I want an easier way to do X” instead of “add Y”)
[comment]: # (Read https://github.com/mozilla-mobile/fenix#i-want-to-file-an-issue for more information)

### What is the user problem or growth opportunity you want to see solved?
Currently to access add on page one has to do multiple clicks
This can be solved by adding a general add on icon in address bar itself. By truncating the full url to just the domain name there will be too much space left which can be utilized to add the add on button and possibly a refresh button

### How do you know that this problem exists today? Why is this important?
Currently to access add on page one has to do many clicks

### Who will benefit from it?
All users 

In short 
Truncate the url to just the domain name and then utilize the space to add the button 
![Screenshot_20210208-100658436 (1)](https://user-images.githubusercontent.com/58127841/107176491-8c7f7780-69f5-11eb-849b-0e8125b6fe24.jpg)
![Screenshot_20210208-100643307 (1)](https://user-images.githubusercontent.com/58127841/107176496-91442b80-69f5-11eb-8431-a83ae338f6dc.jpg)

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/FNXV2-9654)

**Comment by csadilek** [comment]
Moved to bugzilla: https://bugzilla.mozilla.org/show_bug.cgi?id=1809244

Change performed by the [Move to Bugzilla add-on](https://addons.mozilla.org/en-US/firefox/addon/move-to-bugzilla/).
